node/pkg/adminrpc/adminserver_test.go

```diff
@@ -317,3 +320,79 @@ func Test_adminCommands(t *testing.T) {
 		})
 	}
 }
+
+func newNodePrivilegedServiceForGovernorTests() *nodePrivilegedService {
+	gov := governor.NewChainGovernor(zap.NewNop(), &db.MockGovernorDB{}, wh_common.GoTest)
+
+	return &nodePrivilegedService{
+		db:              nil,
+		injectC:         nil,
+		obsvReqSendC:    nil,
+		logger:          nil,
+		signedInC:       nil,
+		governor:        gov,
+		evmConnector:    nil,
+		gk:              nil,
+		guardianAddress: common.Address{},
+	}
+}
+
+func TestChainGovernorResetReleaseTimer(t *testing.T) {
+	service := newNodePrivilegedServiceForGovernorTests()
+
+	// governor has no VAAs enqueued, so if we receive this error we know the input validation passed
+	success := `vaa not found in the pending list`
+	boundsCheckFailure := `the specified number of days falls outside the range of 1 to 7`
+	vaaIdLengthFailure := `the VAA id must be specified as "chainId/emitterAddress/seqNum"`
+
+	tests := map[string]struct {
+		vaaId          string
+		numDays        uint32
+		expectedResult string
+	}{
+		"EmptyVaaId": {
+			vaaId:          "",
+			numDays:        1,
+			expectedResult: vaaIdLengthFailure,
+		},
+		"NumDaysEqualsLowerBoundary": {
+			vaaId:          "valid",
+			numDays:        1,
+			expectedResult: success,
+		},
+		"NumDaysLowerThanLowerBoundary": {
+			vaaId:          "valid",
+			numDays:        0,
+			expectedResult: boundsCheckFailure,
+		},
+		"NumDaysEqualsUpperBoundary": {
+			vaaId:          "valid",
+			numDays:        7,
```

Comment:
These tests could use `maxResetReleaseTimerDays`, `maxResetReleaseTimerDays + 1` and `maxResetReleaseTimerDays + 1000`.

Reply:
Done!